src/main/java/com/cleancommunity/user/UserFactory.java

```diff
@@ -1,10 +1,22 @@
 package com.cleancommunity.user;
 
 public class UserFactory {
-    private int numInstancesMade;   // Why tho
+    private int numInstancesMade;
+    private UserDAO userDAO;
 
-    public static User createUserByRequest(String request){
-        return (1 > 0) ? new Volunteer() : new Admin();
+    public UserFactory() {
+        // TODO Query database to get number of postings instead
```

Comment:
What will postings have to do with creating a User?

Reply:
Oops, stray comment. Will make sure this gets out in future revisions.